client/src/view/Chat/Chat.svelte

```diff
@@ -1,46 +1,31 @@
 <script lang="ts">
-  import HelpImage from '../../lib/assets/icons/help.png'
-  import Button from '../../lib/components/Button.svelte'
-  import StatusDot from './lib/StatusDot.svelte'
+  import { onDestroy, onMount } from 'svelte'
+  import { getSocket } from '../../socket'
+  import ChatView from './ChatView.svelte'
+  import NearbyUsersList from './NearbyUsersList.svelte'
+
+  const socket = getSocket(null)
+
+  let gridUsers: string[] = []
+  let chattingWith: string = ''
+  let chats: { [key: string]: { content: string; owner: string }[] } = {}
```

Comment:
If the component is destroyed (unmounts), it'll delete all the chats. Soon I'll move the chats to local storage.